packages/amplication-client/src/Resource/git/GitActions/GitProviderConnectionList.tsx

```diff
@@ -87,6 +90,11 @@ export const GitProviderConnectionList: React.FC<Props> = ({
         onSyncNewGitOrganizationClick={handleAddProvider}
         disabled={!showBitbucketConnect.hasAccess}
       />
+      <GitProviderConnection
+        provider={EnumGitProvider.AwsCodeCommit}
+        onSyncNewGitOrganizationClick={handleAddProvider}
```

Comment:
Does it work? I don't see that you changed or added something related to this

Reply:
let me remove this for now so it would be just a placeholder to start with